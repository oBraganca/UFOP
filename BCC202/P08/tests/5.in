3
4
luis 26
maria 40
antonio 23
beatriz 19
5
laura 21
ana paula 23
ana julia 22
luisa 22
nadia 27
4
paulo 28
vitor 23
vanessa 29
bruna 32
