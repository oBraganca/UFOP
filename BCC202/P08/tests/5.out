Teste 1:
antonio 23
beatriz 19
luis 26
maria 40

Teste 2:
ana julia 22
ana paula 23
laura 21
luisa 22
nadia 27

Teste 3:
bruna 32
paulo 28
vanessa 29
vitor 23
